src/utils/index.js

```diff
@@ -7,12 +7,24 @@ dotenv.config();
  * @return {string} generated token that is a string data type.
  */
 export const emailVerifyToken = (id) => new Promise((resolve) => {
-  const token = jwt.sign({ id }, process.env.SECRET_KEY_EMAIL_VERIFY_TOKEN, { expiresIn: '24h' });
+  const token = jwt.sign({ id }, process.env.EMAIL_VERIFY_TOKEN_SECRET_KEY, { expiresIn: '24h' });
   if (token) return resolve(token);
 });
+
 /**
- * @param {array} array of data to test
- * @param {element} element of that array
- * @return {integer} index of the  element in array
+ * @param {id} id of newly registered user from the user Table.
+ * @return {string} generated token that is a string data type.
  */
-export const arrayTest = (array, element) => array.indexOf(element);
+export const jwtSignUser = (id) => new Promise((resolve, reject) => {
+  const token = jwt.sign({ id }, process.env.SECRET_KEY_SIGN_UP);
+  if (token) return resolve(token);
+  const ERROR = 'Token cannot be assigned';
```

Comment:
A more descriptive error as to why the token cannot be verified. e.g. Invalid Token

Reply:
Aiit